test/runtests.jl

```diff
@@ -17,4 +22,16 @@ using Test
     @test generate_mesh(control_file, verbose=true) isa String
   end
 
+  @testset "generate_mesh() with ABAQUS output" begin
+    control_file = joinpath(HOHQMesh.examples_dir(), "ABAQUS_IceCreamCone.control")
+    generate_mesh(control_file)
+    parse_mesh = abaqus_read_mesh(joinpath(outdir, "ABAQUS_IceCreamCone.inp"))
+    # set some reference values for comparison. These are the corner IDs for element 114
+    ref_IDs = [140, 141, 154, 153]
+    @test sum(parse_mesh["elements"][114] - ref_IDs) == 0
```

Comment:
Why the sum? Why not just 
```suggestion
    @test parse_mesh["elements"][114] == ref_IDs
```
?

Reply:
That is probably better than relying on exact integer subtraction. This compares a vector so it would return a vector of `true`. So I think I might need to use `all` or something similar

Reply:
No, comparing two vectors for equality yields a scalar. Only if you use `.==` you get a vectorial result.